lib/migration.js

```diff
@@ -481,7 +524,7 @@ function mixinMigration(PostgreSQL) {
       default:
       case 'String':
       case 'JSON':
-        return 'TEXT';
+      case 'Uuid':
```

Comment:
Hmm, what's the difference between `Uuid` and `UUID`?

Reply:
I am following the pattern with other type such as `String`, `Number`. But I think the case doesn't matter. I can change it to `uuid`.
As for the `UUID` below, it's modified to upper case so I use `UUID` there.

Reply:
oh I see, the code comparison gave me an impression that they are two cases in the same function, while actually not lol. 👍